kubernetes/opentelemetry-demo.yaml

```diff
@@ -234,16 +114,9 @@ data:
           from_attribute: k8s.pod.uid
           key: service.instance.id
       resourcedetection:
-        detectors: [gcp]
+        detectors:
+        - gcp
         timeout: 10s
-
-      transform:
-        metric_statements:
-        - context: metric
-          statements:
-          - set(description, "") where name == "queueSize"
```

Comment:
interesting that these are dropped

Reply:
I believe the bugs they were working around are now fixed.